public_html/layouts/basic/modules/Vtiger/resources/ListSearch.js

```diff
@@ -135,6 +135,12 @@ jQuery.Class(
 		registerDateTimeListSearch: function(container) {
 			App.Fields.DateTime.register(this.getContainer());
 		},
+		registerTreeSearch: function() {
+			App.Fields.Tree.register(this.container);
+			let vtigerEditInstance = new Vtiger_Edit_Js();
+			vtigerEditInstance.registerTreeAutoCompleteFields(this.container);
```

Comment:
move registerTreeAutoCompleteFields, registerClearTreeSelectionEvent to App.Fields.Tree